### ISSUE TITLE
implement `getMyAgent` query, and backend zome fn

### ISSUE THREAD

**Issue by Connoropolous** [holo-rea/holo-rea#303]
This depends on #302 

Do a lookup from the current users AgentPubKey via `get_links` to any related Agent record. Just take the first one and return it. We will naively assume one-key one-person for now

**Comment by weswalla** [comment]
currently blocked until we determine if it returns a single agent entry, or a vec, which may require changes to vf spec

**Comment by pospi** [comment]
Single agent entry please! We discussed inferring this from the user's local sourcechain and picking only the latest linked entry, which will resolve any duplicate link issues.

This would be a totally different logic to #309, but that is not needed for MMR.

**Comment by pospi** [comment]
Thinking more about it, to prepare for #309 it would be best to add some validation to ensure that only a single link from an `AgentPubKey` to a profile can be made, and add logic to delete any existing links before assignment. Such validation should be safe to add for well-intentioned agents since all data necessary to make the check will exist on the local sourcechain.
